release-bintray.gradle

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+MIT License
+
+Copyright (c) 2017 Quang Nguyen
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+ */
+
+
+/**
+ * Bintray release configuration. It is adapted from
+ * https://github.com/quangctkm9207/multi-library-bintray/blob/master/release-bintray.gradle
+ */
+ext {
+  bintrayRepo = 'maven' //'test-repo'
+  bintrayName = 'photon-ml'
+  bintrayOrg = 'linkedin'
+
+  publishedGroupId = 'com.linkedin.photon.ml'
+  libraryName = 'photon-ml'
+  artifact = POM_ARTIFACT_ID
+  packagingType = POM_PACKAGING
+  libraryDescription = 'Photon Machine Learning (Photon ML)'
+
+  siteUrl = 'https://github.com/linkedin/photon-ml'
+  gitUrl = 'https://github.com/linkedin/photon-ml.git'
+
+  libraryVersion = '19.0.0'
+
+  licenseName = 'Apache-2.0'
```

Comment:
Is this the original license? Linkedin has moved to BSD-2 license. but we don't need to change here, I suppose. https://github.com/linkedin/detext/blob/master/LICENSE.md

Reply:
We have decided to stick to the original photon-ml license, Apache 2.0.